deploy/role.yaml

```diff
@@ -50,5 +50,6 @@ rules:
   - alert.monstorak.org
   resources:
   - '*'
+  - cephalerts
```

Comment:
I feel we discussed earlier that this operator could be generic one we ideally should not use ceph/gluster etc at CRD defintiion level. IIUC this is generic portion, so suggesting this. Once we need storage system specific code, we can have packages defined accordingly.

Reply:
Taken into account. Resolved.